### PR TITLE
test | GIDK-1h9822 | Aditi | clear button ui fixed

### PR DESCRIPTION
**Pull request by aditi-tiwari17**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 1]
```diff
@@ -399,15 +399,18 @@ button.btn.btn-default.ladda-button {
     height: 36px;
 }
 .clear-filter {
-    button.btn.btn-default {
+    button.btn.btn-filter {
+        background: #e4e4e4;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:403]: use color var

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 2]
```diff
@@ -399,15 +399,18 @@ button.btn.btn-default.ladda-button {
...
+        border: 1px solid #e4e4e4;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:404]: use color var

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 3]
```diff
@@ -399,15 +399,18 @@ button.btn.btn-default.ladda-button {
...
         font-size: 14px;
+        color: #1a237e;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:406]: use color var

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by shalinee-walkover — APPROVED** [review]
Approve

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**